Modules/Microsoft365DSC/DSCResources/MSFT_TeamsCallQueue/MSFT_TeamsCallQueue.psm1

```diff
@@ -52,10 +52,44 @@ function Get-TargetResource
         [System.String]
         $TimeoutAction,
 
+        [Parameter()]
+        [ValidateSet("Queue","Forward","Disconnect","Forward","Voicemail","SharedVoicemail")]
```

Comment:
In the PR title I see that the 2nd Forward probably should be combined with the Voicemail string that is immediately behind it, so ForwardVoicemail instead of Forward, Voicemail.